### ISSUE TITLE
[Bug] Dalle-3 无法使用

### ISSUE THREAD

**Issue by cookieY** [lobehub/lobe-chat#4299]
### 📦 Environment

Docker

### 📌 Version

v1.21.8

### 💻 Operating System

Other Linux

### 🌐 Browser

Chrome

### 🐛 Bug Description

使用 text2image 功能生成图片后 图片无法显示

接口 /webapi/proxy 报错 400 {"error":"Not support internal host proxy"}

后端日志报错

```
I: request to https://oaidalleapiprodscus.blob.core.windows.net/private/org-qCe6JgaC2AUJAAIvDJTOQP7V/user-FpTEyVidRobRD1Pn4XvL5EIM/img-z4iqykp2BxFaZn5hU7hMzPcU.png?st=2024-10-09T01%3A30%3A11Z&se=2024-10-09T03%3A30%3A11Z&sp=r&sv=2024-08-04&sr=b&rscd=inline&rsct=image/png&skoid=d505667d-d6c1-4a0a-bac7-5c84a87759f8&sktid=a48cca56-e6da-484e-a814-9c849652bcb3&skt=2024-10-08T05%3A58%3A53Z&ske=2024-10-09T05%3A58%3A53Z&sks=b&skv=2024-08-04&sig=x9oepvxRk7vTi94h1nO%2BYxOzBp7mVRvmWCrsKv71kMU%3D failed, reason: DNS lookup 224.0.0.3(family:4, host:oaidalleapiprodscus.blob.core.windows.net) is not allowed. Because, It is private IP address.

```

**Comment by xingfeng7788** [comment]
遇到同样的问题，图片链接打开后可以看到图片

**Comment by IvanLi-CN** [comment]
I have a similar problem, I use a global proxy in Fake-IP mode and the IP address resolves to `198.18.0.1/16`.

This is probably the problem introduced by #4044.

**Comment by arvinxx** [comment]
it's a safety issue: https://github.com/lobehub/lobe-chat/security/advisories/GHSA-3fc8-2r3f-8wrg

>  I use a global proxy in Fake-IP mode and the IP address resolves to 198.18.0.1/16.

you need to find a way out I think.
